### PR TITLE
DOCS Some updates to release instructions

### PR DESCRIPTION
**Pull request by hoodmane**

### CODE REVIEW

File: docs/development/maintainers.md [hunk 1]
```diff
@@ -103,21 +109,19 @@ This can be done with either,
...
-the npm package the alpha should have version in the format `x.x.x-alpha.0`. For
-the node package make sure to use `npm publish --tag next` to avoid setting the
-alpha version as the stable release.
-
-If you accidentally publish the alpha release over the stable `latest` tag, you
-can fix it with: `npm dist-tag add pyodide@a.b.c latest` where `a.b.c` should be
-the latest stable version. Then use
-`npm dist-tag add pyodide@a.b.c-alpha.d next` to set the `next` tag to point to the
-just-published alpha release.
+Name the first alpha release `x.x.xa0` and in subsequent alphas increment the
```
Comment (ryanking13) [inline docs/development/maintainers.md:116]: It seems like we were using `a1` for the first alpha release, which, I guess, is not very important.

File: docs/development/maintainers.md [hunk 2]
```diff
@@ -33,53 +32,60 @@ the latest release branch named `stable` (due to ReadTheDocs constraints).
...
      ```
      where `LAST_TAG` is the tag for the last release.
-     Merge the PR.
 
-3. Assuming the upstream `stable` branch exists, rename it to a release branch
-   for the previous major version. For instance if last release was, `0.20.0`,
-   the corresponding release branch would be `0.20.X`,
+3. Make a PR with the updates from steps 1 and 2. Merge the PR.
+
+4. (Major release only.) Assuming the upstream `stable` branch exists,
```
Comment (ryanking13) [inline docs/development/maintainers.md:38]: I was always confused about this. Our up-to-date branch is `main`, not `stable`. Shouldn't there be a command that merges main to stable?

**Review by ryanking13 — APPROVED** [review]
Thanks, @hoodmane!

File: docs/development/maintainers.md (hunk 2 above)
Comment (hoodmane) [reply docs/development/maintainers.md:38]: When we make a major release, say v0.21.0 we do
```
git branch v0.20.X stable
git branch -f stable v0.21.0
# remove branch protection on stable branch
git push -f upstream stable 
```

File: docs/development/maintainers.md (hunk 1 above)
Comment (hoodmane) [reply docs/development/maintainers.md:116]:
```suggestion
Name the first alpha release `x.x.xa1` and in subsequent alphas increment the
```

**Pull request merged**